index.js

```diff
@@ -105,7 +106,12 @@ class Ky {
 			this._options.prefixUrl += '/';
 		}
 
-		this._input = this._options.prefixUrl + this._input;
+		const url = new URL(this._options.prefixUrl + this._input);
+		if (searchParams) {
+			url.search = new URLSearchParams(searchParams).toString();
```

Comment:
Create a new URLSearchParams instance only if searchParams is an object or an array. Don't parse strings when you want a string.